middleware/verifyUser.js

```diff
@@ -0,0 +1,12 @@
+import { User } from '../models';
+
+export default (req, res, next) => User.findById(req.user.id, { where: { id: req.user.id } })
```

Comment:
What if we make this a class method for the `User` model? I learned recently that it's cool to have fat models and thin controllers, and that databases would run things faster. We can just add a `checkVerificationStatus` method that returns if the user is verified or not. Then in this middleware, we can have an async function that'll run the `checkVerificationSatatus` then return your error message or `next()`.